### PR TITLE
chore: prepare Tokio v1.24.0 release

### PR DESCRIPTION
**Pull request by carllerche**

### CODE REVIEW

File: tokio/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+ - rt: add configuration option for max number of I/O events polled from the OS
+   per tick (#5186)
+ - rt: add an environment variable for configuring the default number of worker
+   threads per runtime instance ([#4250])
+
+### Changed
+ - sync: reduce MPSC channel stack usage ([#5294])
+ - io: reduce lock contention in I/O operations  ([#5300])
+ - fs: speed up `read_dir()` by chunking operations ([#5309])
+ - rt: use internal `ThreadId` implementation ([#5329])
```
Comment (Darksonn) [inline tokio/CHANGELOG.md:16]: You are missing #5115.

File: tokio/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
+# 1.24.0 (January 5, 2022)
+
+### Fixed
+ - rt: improve native `AtomicU64` support detection ([#5284])
+
+### Added
+ - rt: add configuration option for max number of I/O events polled from the OS
+   per tick (#5186)
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:8]:
```suggestion
   per tick ([#5186])
```

Comment (carllerche) [reply tokio/CHANGELOG.md]: Thanks, I applied this change in the latest commit.

File: tokio/CHANGELOG.md (hunk 1 above)
Comment (carllerche) [reply tokio/CHANGELOG.md]: Added.

File: tokio/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
+# 1.24.0 (January 5, 2022)
+
+### Fixed
+ - rt: improve native `AtomicU64` support detection ([#5284])
+
+### Added
+ - rt: add configuration option for max number of I/O events polled from the OS
+   per tick ([#5186])
+ - rt: add an environment variable for configuring the default number of worker
+   threads per runtime instance ([#4250])
```
Comment (Darksonn) [inline tokio/CHANGELOG.md:10]: We don't usually have newlines within individual entries in the changelog.

**Review by Darksonn — APPROVED** [review]
LGTM.

File: tokio/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+ - test: don't auto-advance time when a `spawn_blocking` task is running ([#5115])
+
+[#5186]: https://github.com/tokio-rs/tokio/pull/5186
+[#5294]: https://github.com/tokio-rs/tokio/pull/5294
+[#5284]: https://github.com/tokio-rs/tokio/pull/5284
+[#4250]: https://github.com/tokio-rs/tokio/pull/4250
+[#5300]: https://github.com/tokio-rs/tokio/pull/5300
+[#5329]: https://github.com/tokio-rs/tokio/pull/5329
+[#5115]: https://github.com/tokio-rs/tokio/pull/5115
+
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:26]:
```suggestion
[#5309]: https://github.com/tokio-rs/tokio/pull/5309

```

File: tokio/CHANGELOG.md (hunk 3 above)
Comment (carllerche) [reply tokio/CHANGELOG.md:10]: We probably should if the line gets too long. Markdown formatters will insert new lines.

**Review by taiki-e — APPROVED** [review]

**Pull request merged**